### DISCUSSION TITLE
variation of translation according to gender

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#8478] [Q&A]
@Frenzie Somewhere you had told me that it is possible to translate the same word differently in different cases, depending on the gender of the word specified or the command, but I can't find where. Could you explain the process again, please?

Thank you!

**Comment by Frenzie** [comment]
Presumably you're referring to context? http://koreader.rocks/doc/modules/gettext.html#pgettext

**Comment by ichnilatis-gr** [comment]
No, I'm talking about Weblate. You had told me that different translations are possible for one word/phrase, depending on the meaning they have or the word they specify each time. How could I activate this feature?

**Reply by Frenzie** [reply]
That's only possible with a context. Where would you like one?

**Reply by ichnilatis-gr** [reply]
I give you a table with the words. I hope I'm not putting you to any trouble.

For the word **"none"**. I would like a differentiation for the cases: "Word Expansion" / "page crop", "Page Gap"

**"auto"**: "page crop", "word gap" / "Font Hinting"

**"small", "medium", "large"**: "Word Spacing", "Margin", "Line Spacing" "Page Gap" / "Word Gap"
